### DISCUSSION TITLE
How to see the keymaping like in the readme example

### DISCUSSION THREAD

**Discussion by end3r-man** [barry-ran/QtScrcpy#1052] [General]
hey, in the readme example screenshot they show all the keymapping is their any shortcut to do that??

**Comment by w4po** [comment]
Hi, Sorry for the late response but I've just made a visual key mapper for QtScrcpy: https://github.com/w4po/ScrcpyKeyMapper

**Reply by end3r-man** [reply]
> Hi, Sorry for the late response but I've just made a visual key mapper for QtScrcpy: https://github.com/w4po/ScrcpyKeyMapper

its really amazing, ill try it

**Comment by barry-ran** [comment]
QuickAssistant is coming https://lrbnfell4p.feishu.cn/docx/YLCmd8rN5oiK1axvjD0cGgjhnOh
